src/style.css

```diff
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.bigForm {
+  margin-top: 20px;
+  margin-bottom: 20px;
+
+}
+
+.search-bar {
+  position: relative;
+  width: 100%;
+}
+
+.iconBig {
+  position: absolute;
+  margin-left: 26px;
+  margin-top: 25px;
+  width: 19px;
+  height: 19px;
```

Comment:
checklist point 2 (icon and icon--small)

Reply:
done